docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -364,40 +439,174 @@ Dagster Cloud currently includes support for four levels of role-based access co
       <td>
         Manage <a href="/dagster-cloud/account/setting-up-alerts">alerts</a>
       </td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
-      <td className="bg-green-50">Y</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
     </tr>
     <tr>
       <td>Edit workspace</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
-      <td className="bg-green-50">Y</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
+      <td className="bg-green-50">✅</td>
     </tr>
     <tr>
       <td>
         <a href="/dagster-cloud/account/authentication">Administer SAML</a>
       </td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
     </tr>
     <tr>
       <td>View usage</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
     </tr>
     <tr>
       <td>Manage billing</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
     </tr>
   </tbody>
 </table>
+
+  </TabItem>
+</TabGroup>
+
+### Assigning user roles
+
+With the exception of the **Organization Admin** role, user roles are set on a per-deployment basis and enforced both in Dagster Cloud and the GraphQL API. Organization Admins have access to the entire organization, including all [full deployments](/dagster-cloud/developing-testing/managing-deployments), [code locations](/dagster-cloud/developing-testing/code-locations), and [Branch Deployments](/dagster-cloud/developing-testing/branch-deployments).
```

Comment:
any risk people might read this sentence and think org admin is not enforced in the graphql api? (i.e. that it just applies to the first part of the sentence)